src/pages/homeCollection/kitCSV.js

```diff
@@ -111,15 +113,36 @@ const modifyKitQueryResults = (kitsData) => {
 
 const getVialTypesMapping = (collectionType, site, tubeId) => {
   if (collectionType === 'home') {
-    return vialMapping[collectionType]?.[site]?.[tubeId] || ['', '', '', ''];
+    return vialMapping[collectionType]?.[site]?.[tubeId] || ["15ml Nalgene jar",	"Crest Alcohol Free",	"Saliva",	"10"];
```

Comment:
Looks like all the values are the same, so you may be able to just return the fallback value
`["15ml Nalgene jar", "Crest Alcohol Free", "Saliva", "10"]`
and remove the vialMapping object.

Reply:
Done